plio/consumers.py

```diff
@@ -0,0 +1,33 @@
+import json
+from asgiref.sync import async_to_sync
+from channels.generic.websocket import WebsocketConsumer
+from users.serializers import UserSerializer
+
+
+class UserConsumer(WebsocketConsumer):
```

Comment:
why should it not be under `users`?

Reply:
you can maybe even extend the `User` model itself?

Reply:
Also, would be good to have a better name - Viewer, Consumer, PlioConsumer - anything really - UserConsumer sounds weird.